tests/key_test.rs

```diff
@@ -121,12 +121,12 @@ parse_dumps!(
     (test_parse_dumps_0, 0, 18_141, 20_998),
     (test_parse_dumps_1, 1, 18_021, 21_000),
     (test_parse_dumps_2, 2, 18_058, 20_999),
-    (test_parse_dumps_3, 3, 18_098, 20_999),
+    (test_parse_dumps_3, 3, 18_097, 20_998),
```

Comment:
These numbers going down is suspicious, this likely means a secpk key that is not being able to verified/work with the impl. Needs checking out

Reply:
Does it mean it's one key that has the problem?

Reply:
Using `dbg!` in [src/packet/single.rs#L192](https://github.com/rpgp/rpgp/blob/master/src/packet/single.rs#L192) shows there's only one _new parsing error_ in the PR.

The error is `InvalidPacketContent(EllipticCurve(Error))` with a `PublicKey` tag and it's corresponding body is `040000000013052b8104000a02074615813d2d31329d51e5a8e7c8eaa65637e9ad5de00ea3fa65b2f1325aced0a527b6d28e1818e1f3d797d37f3c7c8256ff5f711756255a25f422224540966ed708`.

If there's a simple way to decode that hex, we might see exactly what message it is possibly why it fails.